benchmark_scripts/text_generation/config.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Can we pull this whole thing straight into the python script? Or as a separate python script if we thing the separation is important?

I think we would like to avoid json like this generally if possible. We made that call when making the bert config file https://github.com/keras-team/keras-nlp/blob/master/examples/bert/bert_config.py, but I think the same reasoning applied here.

Reply:
Moved it to a separate Python script! Not putting it in the same script because the config is kinda long.